shared/observability/pyproject.toml

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[project]
+name = "apache-airflow-shared-observability"
+description = "Shared observability code for Airflow distributions"
+version = "0.0"
+classifiers = [
+    "Private :: Do Not Upload",
+]
+
+dependencies = [
+    "pendulum>=3.1.0",
+    "structlog>=25.4.0",
+]
+
+[project.optional-dependencies]
+"otel" = [
+    "opentelemetry-api>=1.27.0",
+    "opentelemetry-exporter-otlp>=1.27.0",
+    "opentelemetry-proto<9999,>=1.27.0",
+]
+"statsd" = [
+    "statsd>=3.3.0",
+]
+"datadog" = [
+    "datadog>=0.50.0",
+]
+"all" = ["apache-airflow-shared-observability[otel,statsd,datadog]"]
+
+[dependency-groups]
+dev = [
+    "apache-airflow-devel-common",
+    "apache-airflow-shared-configuration",
```

Comment:
We don't need this?

Reply:
You are right. Tests passed without it. I removed it.